acceptance/tests/pxp-module-puppet/run_puppet_result_changed.rb

```diff
@@ -1,13 +1,14 @@
 require 'pxp-agent/test_helper.rb'
-
-ENVIRONMENT_NAME = 'changed_result'
+require 'puppet/acceptance/environment_utils'
 
 test_name 'C93062 - Run puppet and expect \'changed\' result' do
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  env_name = test_file_name = File.basename(__FILE__, '.*')
+  environment_name = mk_tmp_environment(env_name)
 
   step 'On master, create a new environment that will result in Changed' do
-    environmentpath = master.puppet['environmentpath']
-    site_manifest = "#{environmentpath}/#{ENVIRONMENT_NAME}/manifests/site.pp"
-    on(master, "cp -r #{environmentpath}/production #{environmentpath}/#{ENVIRONMENT_NAME}")
+    site_manifest = "#{environmentpath}/#{environment_name}/manifests/site.pp"
     create_remote_file(master, site_manifest, <<-SITEPP)
 node default {
   notify {'Notify resources cause a Puppet run to have a \\'changed\\' outcome ☃':}
```

Comment:
This has a snow-man in it, so these are both testing non-ASCII. That's what I meant when I said there was an acceptance test (I added the snowman in my PR fixing the issue).